### ISSUE TITLE
Share this app bug

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#186]
The title and the link on the top should be in separate lines.

**Comment by Rajat-Sen** [comment]
Also, on Android it shows these options which leads to errors: bluetooth, currents, drive and hangouts.
